### PR TITLE
Fix exception if bootstrap non-existent store

### PR DESCRIPTION
**Pull request by bkniffler**
Prevent exception if trying to bootstrap with non-existent store.

### CODE REVIEW

File: src/alt.js [hunk 1]
```diff
@@ -226,8 +226,10 @@ const StoreMixinEssentials = {
 const setAppState = (instance, data, onStore) => {
   const obj = JSON.parse(data)
   Object.keys(obj).forEach((key) => {
-    assign(instance.stores[key][STATE_CONTAINER], obj[key])
-    onStore(instance.stores[key])
+    if(instance.stores[key]){
```
Comment (goatslacker) [inline src/alt.js]: nit: whitespace

`if (instance.stores[key]) {`

Comment (bkniffler) [inline src/alt.js]: Done!

**Comment by jdlehman** [comment]
@bkniffler do you want to rebase this with master so we can merge it in? I can also rebase it if you prefer.

**Comment by goatslacker** [comment]
b458450e9c46fe38a44f3a23a6dc0c1e08bbf012

**Pull request closed without merging**